packages/core/src/types.ts

```diff
@@ -1279,30 +1279,39 @@ export type ExtractEvent<
   TEventType extends TEvent['type']
 > = TEvent extends { type: TEventType } ? TEvent : never;
 
-export interface ActorRef<TEvent extends EventObject, TEmitted = any>
-  extends Subscribable<TEmitted> {
+export interface BaseActorRef<TEvent extends EventObject, TEmitted = any> {
   send: Sender<TEvent>;
+  subscribe?: Subscribable<TEmitted>['subscribe'];
```

Comment:
I've seen that you have removed this first but then added it back. Any particular reason? What makes `subscribe` special to be part of this interface (even if it's optional)?

Reply:
It's what makes `TEmitted` relevant; I guess I can just remove it from the type.